src/realm-settings/ECDSAGeneratedModal.tsx

```diff
@@ -36,8 +36,7 @@ export const ECDSAGeneratedModal = ({
   handleModalToggle,
   open,
   refresh,
-}: // save,
-ECDSAGeneratedModalProps) => {
+}: ECDSAGeneratedModalProps) => {
   const { t } = useTranslation("groups");
```

Comment:
This should not be `"groups"`, right?

```suggestion
  const { t } = useTranslation("realm-settings");
```